internal/controller/api/v1beta1/agent_types.go

```diff
@@ -225,6 +223,12 @@ type DebugInfo struct {
 	// EventsURL specifies an HTTP/S URL that contains events which occured during the cluster installation process
 	// +optional
 	EventsURL string `json:"eventsURL,omitempty"`
+	// +optional
+	// Current state of the Agent
+	State string `json:"state,omitempty"`
+	//Additional information pertaining to the status of the Agent
+	// +optional
+	StateInfo string `json:"stateInfo,omitempty"`
```

Comment:
Should align with the actual fields naming of `status` and `status_info`

Reply:
we don't use snake case in CRD field names

Reply:
That's fine. I referred to the use of `Status` instead of `State` terminology.
So it would be accordingly `Status` and `StatusInfo` instead of `State` and `StateInfo`

Reply:
@avishayt @mhrivnak what do you think? 
I think that state and state info are clearer.

Reply:
I agree - state and state info